### DISCUSSION TITLE
WalrusX：基于Sui和Walrus的去中心化X

### DISCUSSION THREAD

**Discussion by JasonRUAN** [creatorsand/.github#121] [Ideas]
## 一句话 IDEA

基于Sui和Walrus的去中心化X。

## 我的角色

设计+开发

## 社群昵称

rzexin

## 个人简介

多年Web2的后端开发者，前端初学者，希望基于Sui和Walrus做出点有趣的东西。

## 产品是给谁用的？

热爱分享的朋友

**Comment by v1xingyue** [comment]
ai打分

[a comment by an automated account is not shown]
